apps/videos/urls.py

```
@@ -1,8 +1,7 @@
 # apps/videos/urls.py
-from django.urls import path
-from .views import UserVideoListView, VideoDeleteView
+from .views import UserVideoListView, VideoDeleteView, VideoDetailView
 
 urlpatterns = [
     path('', UserVideoListView.as_view(), name='user-video-list'),
```

Ruff (0.12.2)

Undefined name `path`

(F821)